service/gradle.lockfile

```diff
@@ -94,7 +94,8 @@ com.google.api:api-common:2.1.5=compileClasspath,productionRuntimeClasspath,runt
 com.google.api:gax-grpc:2.13.0=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
 com.google.api:gax-httpjson:0.98.0=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
 com.google.api:gax:2.13.0=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
-com.google.apis:google-api-services-bigquery:v2-rev20201030-1.30.10=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+com.google.apis:google-api-services-bigquery:v2-rev20201030-1.30.10=compileClasspath,productionRuntimeClasspath,runtimeClasspath
```

Comment:
This is the only comment not done yet, still discussing with Dan

Reply:
Discussed with Dan. So there are 2 version of GCP client libraries ([see here](https://groups.google.com/g/gcd-discuss/c/CkabuD7Xrws/m/hpUYxskKCwAJ)):

`com.google.cloud` - Easier to use, recommended by GCP
`com.google.apis` - CRL uses this, per Dan because they're more consistent

Dan says it's ok for tests to use former, despite CRL using latter